app/views/clients/createclient.html

```diff
@@ -276,7 +276,7 @@
                         </div>
                         <div class="form-group" ng-show="street">
                            <div class="control-label col-sm-2">
-                              <label>{{ 'label.input.street' | translate }}</label>
+                              <label>{{ 'label.input.street' | translate }}<span class="required">*</span></label>
                            </div>
                            <div class="col-sm-3">
                               <input type="text" name="street"  ng-model="addressArray[$index].street"  class="form-control"/>
```

Comment:
Add required and late-validate and form-validate tags to fix this appropriately.